app/models/set_member_subject.rb

```diff
@@ -15,6 +16,7 @@ class SetMemberSubject < ActiveRecord::Base
   can_through_parent :subject_set, :update, :show, :destroy, :index, :update_links,
     :destroy_links
 
+  before_save :timestamp_newly_retired_workflows
```

Comment:
So you'd be like `retired_subject_workflow_counts << non_retired_record` and then leveraging the save functionatliy of the `sms` to retire it?

Just wondering if we should be setting this retired value on the record before adding to the association Or having an explicit method to do it instead of co-opting the save function.